<?xml version="1.0"?>
<launch>  
  <!-- Start simulation environment and controllers -->
  <include file="$(dirname)/system.launch"/>

  <!-- Start controller service -->
  <node pkg="assignment_ur" type="UR5_service" name="ur5_custom_service" output="screen"/>

</launch>